daisy_workflows/build-publish/enterprise_linux/almalinux_9.wf.json

```diff
@@ -23,6 +23,14 @@
     "installer_iso": {
       "Required": true,
       "Description": "The AlmaLinux 9 installer ISO to build from."
+    },
+    "run_sbom_bool": {
+      "Value": "false",
+      "Description": "Determines if the SBOM for the image will be generated"
```

Comment:
I commented this on the other PR, please end the description line with a dot for consistency.

Reply:
done